docs/user/troubleshootingbootstrap.md

```diff
@@ -0,0 +1,220 @@
+# Troubleshooting Bootstrap Failures
+
+Unfortunately, there will always be some cases where OpenShift fails to install properly. In these events, it is helpful to understand the likely failure modes as well as how to troubleshoot the failure.
+
+## Gathering bootstrap failure logs
+
+### Using the installer provisioned workflow
+
+When users are using the installer to create the OpenShift cluster, the installer has all the information to automatically capture the logs from bootstrap host in case of failure.
```

Comment:
Assuming you configured a SSH key and the installer can find the associated private key or SSH agent.  We talk about that a bit [around here][1], and you talk about it a bit more below, but it may be worth qualifying this sentence here...  Or maybe dropping the `#### Authenticating with...` subsection header and just including `The installer will use the user's ...` sentence in this paragraph?

[1]: https://github.com/openshift/installer/blob/afc6983b088e31ce8458543778b89d3fad8ab73c/docs/user/troubleshooting.md#kubernetes-api-is-unavailable

Reply:
> Assuming you configured a SSH key...

Ah, I see I'm stale from the generated-key comments below.